src/scss/components/_leaflet-map.scss

```diff
@@ -2,6 +2,7 @@
 @import 'leaflet-geosearch/dist/geosearch.css';
 
 @import '../mixins/prefix';
+@import 'leaflet-gesture-handling/dist/leaflet-gesture-handling.css';
```

Comment:
please pay attention to the import ordering - you can see that there already is a leaflet-related CSS import at the top. This belongs directly below it (sorting alphabetically).

In general, we follow the pattern: stdlib / third party dependencies / own project / relative imports for the import groups.